app/controllers/competitions_controller.rb

```diff
@@ -842,7 +842,7 @@ def my_competitions
       registrations = current_user.registrations.includes(:competition).accepted.reject { |r| r.competition.results_posted? }
       registrations.concat(current_user.registrations.includes(:competition).pending.select { |r| r.competition.upcoming? })
       @registered_for_by_competition_id = registrations.uniq.to_h do |r|
-        [r.competition.id, r]
+        [r.competition.id, r.as_json({ only: %w[competing_status], include: %w[], methods: %w[] })]
```

Comment:
Uhm, couldn't this be handled by just calling `r.competing_status`?

Reply:
true. Changed!